include/Data/DataGetter.hpp

```diff
@@ -15,7 +16,7 @@ namespace SongDetailsCache {
                 std::shared_ptr<std::vector<uint8_t>> data;
             };
 
-            static const std::unordered_map<std::string, std::string> dataSources;
+            static const std::map<std::string, std::string> dataSources;
```

Comment:
Please keep this an unordered map and just reorder them to fix the order. put the one that should be checked first at the bottom, and last at the top. that's how unordered map works